### ISSUE TITLE
2.0: TypeError: import_htmx.default.config is undefined when defining config programmatically

### ISSUE THREAD

**Issue by danjac** [bigskysoftware/htmx#2657]
This was working in **1.9x**:

```javascript
import htmx from "htmx.org";

// global HTMX configuration
// https://htmx.org/docs/#config

htmx.config.historyCacheSize = 0;
htmx.config.refreshOnHistoryMiss = false;
htmx.config.scrollBehavior = "smooth";
htmx.config.scrollIntoViewOnBoost = false;
htmx.config.useTemplateFragments = true;
```

In **2.0.0** this causes the following error:

**TypeError: import_htmx.default.config is undefined**

The workaround of course is to use the `<meta>` tag, but has this behavior changed in 2.0?

**Comment by danjac** [comment]
I also tried the workaround described here for webpack: https://htmx.org/docs/#webpack however this did not work, same result (note: I'm using esbuild, not webpack).

**Comment by restless** [comment]
Have you seen this: https://github.com/bigskysoftware/htmx/issues/2628?

TL;DR;
Try `import htmx from "htmx.org/dist/htmx.esm";`

**Comment by danjac** [comment]
@restless yes, that appears to work, thanks. FYI if updating the docs for webpack, the solution also works for esbuild.

**Comment by davidlesieur** [comment]
Using Parcel here. `import htmx from "htmx.org/dist/htmx.esm";` works in that case too.

**Comment by 1cg** [comment]
fixed in 2.0
